chaos_kitten/cli.py

```
@@ -175,371 +174,84 @@ def scan(
+            if chaos_findings:
+                console.print(f"   Chaos Mode: {len(chaos_findings)}")
+        else:
+            console.print("\n[bold green]✅ No vulnerabilities found. Your API is a tough kitten![/bold green]")
+
+        # Generate report
+        reporter_cfg = cfg.get("reporting", {})
+        reporter = Reporter(
+            output_path=reporter_cfg.get("output_path", "./reports"),
+            output_format=reporter_cfg.get("format", "html"),
+        )
+        report_file = reporter.generate({"vulnerabilities": vulnerabilities}, target_url)
```

Ruff (0.15.1)

[error] Undefined name `target_url`

(F821)